migrations/Version20201216012133.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+declare(strict_types=1);
+
+namespace DoctrineMigrations;
+
+use Doctrine\DBAL\Schema\Schema;
+use Doctrine\Migrations\AbstractMigration;
+
+/**
+ * Auto-generated Migration: Please modify to your needs!
+ */
+final class Version20201216012133 extends AbstractMigration
+{
+    public function getDescription() : string
+    {
+        return '';
+    }
+
+    public function up(Schema $schema) : void
+    {
+        // this up() migration is auto-generated, please modify it to your needs
```

Comment:
Remove auto comments?

Reply:
MusikAnimal is a Wikipedophile.  I hate Wikipedia Administrators who keep blocking me.  I have every right to vandalise Wikipedia and you have no right to stop me.  ON WHEELS